src/conf.c

```diff
@@ -59,7 +59,9 @@ static char conf_vars[MAX_CONF_VARS][MAXBUF] = {
 	"USE_TLS",
 	"SMTP_AUTH_USER",
 	"SMTP_AUTH_PASS",
-	"VCARD"
+	"VCARD",
+	"TIMEOUT",
+	""
```

Comment:
Would it make sense to just kill the MAX_CONF_VARS macro all together and have `conf_vars` just be a pointer-pointer?

    static char **conf_vars = {
            "USE_TLS",
            /* ... */,
            "TIMEOUT",
            NULL
    }